__tests__/UrlBuilder.test.ts

```diff
@@ -2,31 +2,44 @@ import { UrlBuilder } from '../src/UrlBuilder';
 
 describe('UrlBuilder: ', () => {
   const config = {
-    issuer: 'http://localhost:8081',
+    authUrl: 'http://localhost:8081',
     appId: '59fd884d8f6b180001f5b4e2',
     scopes: [],
-    redirectUri: 'http://localhost:3000',
+    returnTo: 'http://localhost:8081/returnTo',
```

Comment:
what is the difference between `returnTo` and `callbackUrl`. I believe we need to improve naming here.